apertium-eng-spa.eng-spa.lrx

```diff
@@ -8715,7 +8715,185 @@ convertirse en un personaje de moda o importante. -->
+        <match lemma="social" tags="adj"/>
+        <match lemma="economic" tags="adj"/>
+        <match lemma="academic" tags="adj"/>
+      </or>
+      <match lemma="performance" tags="n.*">
+        <select lemma="rendimiento" tags="n.*"/>
+      </match>
+    </rule>
+
+    <!--
+      eng: from a letter dated
+      spa: de una carta datada de
+    -->
+    <rule weight="1.0">
+      <or>
+        <match lemma="from" tags="pr"/>
+        <match lemma="of" tags="pr"/>
+        <match lemma="in" tags="pr"/>
+      </or>
+      <match lemma="a" tags="det.ind.*"/>
+      <match lemma="letter" tags="n.*">
+        <select lemma="carta" tags="n.*"/>
+      </match>
+      <or>
+        <match lemma="date" tags="vblex.pp"/>
+        <match lemma="find" tags="vblex.pp"/>
+        <match lemma="write" tags="vblex.pp"/>
+      </or>
+    </rule>
+
+    <!--
+      eng: draft of the article
+      spa: borrador del artículo
+    -->
+    <rule weight="1.0">
+      <or>
+        <match lemma="draft" tags="n.*"/>
+        <match lemma="version" tags="n.*"/> <!-- e.g "final version of the article" -->
+      </or>
+      <match lemma="of" tags="pr"/>
+      <or>
+        <match lemma="the" tags="det.def.*"/>
+        <match lemma="a" tags="det.ind.*"/>
+      </or>
+      <match lemma="article" tags="n.*">
+        <select lemma="artículo" tags="n.*"/>
+      </match>
+    </rule>
+
+    <!-- 
+      eng: use the term
+      spa: usar/emplear el término
+    -->
+    <rule weight="1.0">
+      <or>
+        <match lemma="use" tags="vblex.*"/>
+        <match lemma="apply" tags="vblex.*"/>
+        <match lemma="employ" tags="vblex.*"/>
+      </or>
+      <or>
+        <match lemma="the" tags="det.def.*"/>
+        <match lemma="a" tags="det.ind.*"/>
+      </or>
+      <match lemma="term" tags="n.*">
+        <select lemma="término" tags="n.*"/>
+      </match>
+    </rule>
+
+    <!--
+      eng: [prpers/np] appears
+      spa: [prpers/np] aparece
+    -->
+    <rule weight="1.0">
+      <or>
+        <match lemma="prpers" tags="prn.subj.*"/>
+        <match tags="np.*"/>
+      </or>
+      <match lemma="appear" tags="vblex.*">
+        <select lemma="aparecer" tags="vblex.*"/>
+      </match>
+    </rule>
+
+    <!--
+      eng: collection of independent states
+      spa: grupo de estados independientes
+    -->
+    <rule weight="1.0">
+      <match lemma="collection" tags="n.*">
+        <select lemma="grupo" tags="n.*"/>
+      </match>
+      <match lemma="of" tags="pr"/>
+      <match lemma="independent" tags="adj"/>
+      <or>
+        <match lemma="state" tags="n.pl"/>
+        <match lemma="country" tags="n.pl"/>
+        <match lemma="nation" tags="n.pl"/>
+        <match lemma="region" tags="n.pl"/>
+        <match lemma="province" tags="n.pl"/>
+      </or>
+    </rule>
+    
+    <!--
+      eng: over the year
+      spa: durante el año
+    -->
+    <rule weight="1.0">
+      <match lemma="over" tags="pr">
```

Comment:
Isn't it rather "a lo largo de los años"?

Reply:
In this case, `eng-spa-biltrans` only shows _sobre_ and _durante_ as possible translations of _over_.

That "a lo largo de los años" might be a thing if _year_ is in plural, something that I haven't noticed though. Maybe I could change it so that it only matches _year_ just when it's singular?

Reply:
Can't one add a multiword translation in the dictionary just to be selected by this rule?
Does the output of <select> really need to match the dictionary or can it be anything?

Reply:
Lexical selection must choose from among the outputs of the dictionary, but yes, multiwords are definitely an option that I forgot about when making my earlier suggestions.

Reply:
Alright, so if you want, I can add that multiword translation.

Reply:
Approval won't depend on that, I guess.